eras/conway/impl/testlib/Test/Cardano/Ledger/Conway/Imp/GovSpec.hs

```diff
@@ -81,6 +83,123 @@ spec =
+              , pProcDeposit = pp ^. ppGovActionDepositL
+              , pProcAnchor = def
+              }
+        submitYesVote_ (StakePoolVoter khSPO) gaidNoConf
+        submitYesVote_ (DRepVoter $ KeyHashObj drep) gaidNoConf
+        replicateM_ 4 passEpoch
+        assertNoCommittee
+      it "SPO needs to vote on security-relevant parameter changes" $ do
+        (drep, ccCred) <- electBasicCommittee
+        khPool <- setupPoolWithStake $ Coin 42_000_000
+        gaidThreshold <- do
+          pp <- getsNES $ nesEsL . curPParamsEpochStateL
+          (pp ^. ppPoolVotingThresholdsL . pvtSecurityRelevantL) `shouldBe` minBound
+          rew <- registerRewardAccount
+          let ppUpdate =
+                emptyPParamsUpdate
+                  & ppuPoolVotingThresholdsL
+                    .~ SJust
+                      PoolVotingThresholds
+                        { pvtSecurityRelevant = 1 %! 2
+                        , pvtMotionNoConfidence = 1 %! 2
+                        , pvtHardForkInitiation = 1 %! 2
+                        , pvtCommitteeNormal = 1 %! 2
+                        , pvtCommitteeNoConfidence = 1 %! 2
+                        }
+                  & ppuGovActionLifetimeL .~ SJust (EpochInterval 100)
+          gaidThreshold <-
+            submitProposal $
+              ProposalProcedure
+                { pProcReturnAddr = rew
+                , pProcGovAction = ParameterChange SNothing ppUpdate SNothing
+                , pProcDeposit = pp ^. ppGovActionDepositL
+                , pProcAnchor = def
+                }
+          submitYesVote_ (DRepVoter drep) gaidThreshold
+          submitYesVote_ (CommitteeVoter ccCred) gaidThreshold
+          logAcceptedRatio gaidThreshold
+          pure gaidThreshold
+        passEpoch
+        logAcceptedRatio gaidThreshold
+        passEpoch
+        gaidMinFee <- do
+          pp <- getsNES $ nesEsL . curPParamsEpochStateL
+          (pp ^. ppPoolVotingThresholdsL . pvtSecurityRelevantL) `shouldBe` (1 %! 2)
+          rew <- registerRewardAccount
+          gaidMinFee <-
+            submitProposal $
+              ProposalProcedure
+                { pProcReturnAddr = rew
+                , pProcGovAction =
+                    ParameterChange
+                      (SJust (PrevGovActionId gaidThreshold))
+                      ( emptyPParamsUpdate
+                          & ppuMinFeeAL .~ SJust (Coin 12_345)
+                      )
+                      SNothing
+                , pProcDeposit = pp ^. ppGovActionDepositL
+                , pProcAnchor = def
+                }
+          submitYesVote_ (DRepVoter drep) gaidMinFee
+          submitYesVote_ (CommitteeVoter ccCred) gaidMinFee
+          pure gaidMinFee
+        passEpoch
+        logAcceptedRatio gaidMinFee
+        passEpoch
+        do
+          pp <- getsNES $ nesEsL . curPParamsEpochStateL
+          (pp ^. ppMinFeeAL) `shouldBe` Coin 44
```

Comment:
This locks us in into the fact that initial value is `44`. We should avoid hard coding values like this. It is better to record the original value for the `initMinFeeA` at the beginning of the test.

```suggestion
          (pp ^. ppMinFeeAL) `shouldBe` initMinFeeA
```